workers/data_refinery_workers/processors/test_no_op.py

```diff
@@ -14,6 +17,62 @@
     Sample,
 )
 from data_refinery_workers.processors import no_op
+from data_refinery_workers.processors.testing_utils import assertMostlyAgrees
+
+
+def prepare_job(job_info: dict) -> ProcessorJob:
+    job = ProcessorJob()
+    job.pipeline_applied = "NO_OP"
+    job.save()
+
+    og_file = OriginalFile()
+    og_file.source_filename = job_info["source_filename"]
+    og_file.filename = job_info["filename"]
+    og_file.absolute_file_path = job_info["absolute_file_path"]
+    og_file.is_downloaded = True
+    og_file.save()
+
+    sample = Sample()
+    sample.accession_code = job_info["accession_code"]
+    sample.title = job_info["accession_code"]
+    sample.platform_accession_code = job_info["platform_accession_code"]
+    man = job_info.get("manufacturer", None)
+    if man is not None:
+        sample.manufacturer = man
+    # The illumina samples need the human organism
+    if job_info.get("manufacturer", None) == "ILLUMINA":
```

Comment:
```suggestion
    if man == "ILLUMINA":
```
I don't love the variable name `man` lol